homeassistant/components/wirelesstag/sensor.py

```diff
@@ -107,6 +152,11 @@ def _sensor(self):
         """Return tag sensor entity."""
         return self._tag.sensor[self._sensor_type]
 
+    @property
+    def _sensor_type(self):
```

Comment:
It's better to make this an instance attribute than a property.

Reply:
ok, changed.